pom.xml

```diff
@@ -108,6 +108,7 @@
         <module>extensions-contrib/parquet-extensions</module>
         <module>extensions-contrib/statsd-emitter</module>
         <module>extensions-contrib/orc-extensions</module>
+        <module>extensions-contrib/scan-query</module>
```

Comment:
Besides orc-extentions, I found more exceptions is the list, I think ordering is not a requirement.